packages/design-system-docs/src/pages/startup/child-design-systems.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Child Design Systems
+weight: 6
+---
+
+The CMS design system (CMSDS) provides a set of components and styles to meet many design and development needs. However, there are cases where product teams need to modify a design system beyond [theming]({{root}}/startup/theming/) and publish a child design system for other product teams to consume.
+
+## What is a child design systems?
+
+A child design system is a design system built off of the core CMSDS for a specific context. It contains custom assets (components, styles, etc..) and is published via NPM to be consumed by multiple product teams. A child design system can contain new components, modifications to existing core components, or modifications to styles. Child design systems import the core CMSDS as a dependency which allows for receiving updates, using developer tools, scripts, and documentation from the core CMSDS to manage their own design system.
+
+## Who should use a child design system?
+
+A common use case for child design systems is when multiple product teams are working under the same CMS site. These separate product teams would share custom assets and code (i.e. brand colors, site header, custom icons, etc), and would benefit from a documentation site to reference as a single source of truth. If you are working as a single team and don't have a need for a documentation site, a child design system is probably not for you.
+
+### Child design system guidelines
+
+- A child design system should be actively maintained by other product teams. The core CMSDS team is not responsible for updating child design systems, however we are responsible for guidelines and assisting with training and adoption.
+- Users of child design system should contribute back to the larger CMS design system community by filing issues/bugs, giving feedback, and general maintenance. Collaboration and participation are essential to the success of the CMS design system family.
+- A child design system should defer to the core CMS design system for best practices or accessibility guidelines.
```

Comment:
It could be helpful to elaborate on what falls under "best practices". The main question I would have after reading this is, in terms of modifications and additions to the CMSDS, how does the CMSDS govern what changes can or can't be made in a child design system and where can I find that information?

Reply:
Maybe adding the workflow diagram for adding a component? 
https://raw.githubusercontent.com/CMSgov/design-system/master/.github/images/CMS-Design-System-component-decision-workflow.jpg 

Alo this image is being updated.

Reply:
This is kind of hard, we can share the flowchart but I don't think its actually very helpful. I think eventually we should create a child design system example that has different types of customizations (overriding colors, adding new `Button` variants, new components, etc). For larger changes, in my mind the best case would be to have these decisions be made collaboratively (aka via discussions on slack). Not sure how to synthesis these into practices however...